src/ol_infrastructure/applications/edxapp/edxapp_policy.hcl

```diff
@@ -0,0 +1,31 @@
+path "mariadb-mitxonline/creds/edxapp" {
+  capabilities = ["read"]
+}
+
+path "mariadb-mitxonline/creds/edxapp-csmh" {
+  capabilities = ["read"]
+}
+
+path "mongodb-mitxonline/creds/app" {
+  capabilities = ["read"]
+}
+
+path "sys/leases/renew" {
+  capabilities = [ "update" ]
+}
+
+path "secret-mitxonline/edxapp" {
```

Comment:
Do we need to add a `*` to those policies as this will only allow read access to that specific path and nothing under it unless we don't plan on having any other subpaths.

Reply:
Yeah, there's no subpath there. That is the full path to a key that contains a JSON object with the needed values.